webapp/app/.server/db/schema.ts

```diff
@@ -151,7 +151,7 @@ export const l2CancellationTypeEnum = z.enum(["ZK_GOV_OPS_GOVERNOR", "ZK_TOKEN_G
 
 export type L2CancellationType = z.infer<typeof l2CancellationTypeEnum>;
 
-export const l2CancellationStatusEnum = z.enum(["ACTIVE", "DONE", "CANCELED"]);
+export const l2CancellationStatusEnum = z.enum(["ACTIVE", "DONE", "NONCE_TOO_LOW", "EXPIRED"]);
```

Comment:
The status can be changed to a more descriptive value. Right now expired could mean that the cancellation expired.
```suggestion
export const l2CancellationStatusEnum = z.enum(["ACTIVE", "DONE", "NONCE_TOO_LOW", "L2_PROPOSAL_EXPIRED"]);
```

Reply:
Done!